crates/orderbook/src/database/orders.rs

```diff
@@ -529,6 +594,32 @@ fn is_buy_order_filled(amount: &BigDecimal, executed_amount: &BigDecimal) -> boo
     !executed_amount.is_zero() && *amount == *executed_amount
 }
 
+type SolverName = String;
+
+#[derive(Serialize, PartialEq, Debug, Clone)]
+#[cfg_attr(any(test, feature = "e2e"), derive(serde::Deserialize))]
+#[serde(tag = "type", rename_all = "camelCase", content = "value")]
+pub enum Status {
```

Comment:
maybe is it worth adding `Expired`?

Reply:
For what purpose and how is it supposed to be obtained since `OrderEventLabel` doesn't track that.

Reply:
From the frontend point of view, it is not needed. But since we have open API, and users potentially will want to fetch the status in their trading bots, `expired` comes quite handy to know the reason of why the order wasn't executed, since they created an order with a short `valid_to`.

Reply:
Seems like this API is going to be heavily used by FE. Fetching `valid_to` requires additional DB query or introducing caching, which doesn't make much sense just for a single `Expired` status where the timestamp can be checked by users or using Explorer, IMO.

Reply:
As Mateo said, not needed by the frontend ATM, and I can't say for other use cases.
Fine by me not having it.